### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
## What?
Landing Page of the Ecommerce App made, with components.
- Landing Page 54b34d1e60728e59c88966001e3659cd78ece78c
    - Navigation Top bea9d191ea4f46755982f9354082b990a0a730a4
    - Hero Section f92399f357e6c761febebf7872c12ce63e8d8804
    - Categories f4897d315eae194c45b39763f62384f88f765519
    - New Arrival 3f172c66e8fd4aa99e96530bde8ff793cdee9187

## Screenshots
temp-heroSection
temp-categories
temp-newArrival

### CODE REVIEW

File: src/pages/landing-page/landing-page.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    padding-bottom: 1rem;
+}
+.category-heading span{
+    color: var(--color-primary);
+}
+
+.category-images{
+    width: 90%;
+    display: flex;
+    flex-direction: row;
```
Comment (sushil-mod) [inline src/pages/landing-page/landing-page.css]: just a suggestion ,flex  has a default value flex-direction : row , you can remove it

**Review by sushil-mod — APPROVED** [review]
Landing page looks nice , just a suggestion ,rest is good

**Review by JantuDeb — APPROVED** [review]
UI looks cool

File: src/pages/landing-page/landing-page.css (hunk 1 above)
Comment (abhijitdotsharma) [reply src/pages/landing-page/landing-page.css]: Yes, I did the changes

**Pull request merged**